.github/workflows/action.yml

```diff
@@ -0,0 +1,34 @@
+name: Satellite-Upgrade
```

Comment:
Optional: Can you use the name Robottelo - CI or Satellite - CI , similar to robottelo/robottelo-ci repos

Reply:
Satellite-Upgrade - CI is more appropriate than rest.